### PR TITLE
AB#69857 withdraw submitted application

### PR DESCRIPTION
**Pull request by dawid-pawulski**

### CODE REVIEW

File: src/HE.InvestmentLoans.WWW/Views/LoanApplicationV2/ApplicationDashboard.cshtml [hunk 1]
```diff
@@ -42,10 +45,10 @@
         </nav>
     </div>
 
-    <div class="govuk-grid-column-two-thirds govuk-!-padding-top-3">
```
Comment (Carq) [inline src/HE.InvestmentLoans.WWW/Views/LoanApplicationV2/ApplicationDashboard.cshtml:45]: Why did you remove padding?

**Review by Carq — APPROVED** [review]

File: src/HE.InvestmentLoans.WWW/Views/LoanApplicationV2/ApplicationDashboard.cshtml (hunk 1 above)
Comment (dawid-pawulski) [reply src/HE.InvestmentLoans.WWW/Views/LoanApplicationV2/ApplicationDashboard.cshtml:45]: ups! mistake during merge. Fix is coming

**Review by burzynskijakub — APPROVED** [review]

**Review by jakub-lotko — APPROVED** [review]
